src/main/java/com/jensongr/songr/controller/AlbumController.java

```diff
@@ -1,20 +1,48 @@
 package com.jensongr.songr.controller;
 
 import com.jensongr.songr.model.Album;
+import com.jensongr.songr.repository.AlbumRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
+import java.util.List;
+
 @Controller
 public class AlbumController {
+
+    private final AlbumRepository albumRepository;
+
+    @Autowired
+    public AlbumController(AlbumRepository albumRepository) {
+        this.albumRepository = albumRepository;
+    }
+
     @GetMapping("/albums")
     public String albums(Model model) {
-        Album[] albums = {
-                new Album("Channel Orange", "Frank Ocean", 17, 62 * 60 + 18, "channel_orange.jpeg"),
-                new Album("AM", "Arctic Monkeys", 12, 41 * 60 + 38, "am.jpg"),
-                new Album("Sawayama", "Rina Sawayama", 13, 43 * 60, "sawayama.png")
-        };
+        List<Album> albums = albumRepository.findAll();
         model.addAttribute("albums", albums);
-        return "album";
+        return "albums";
```

Comment:
Make sure this exactly matches the name of the html template you're trying to return. This will try to return a template called "albums.html", but your template is called "album.html".